slm.toml

```diff
@@ -1,5 +1,5 @@
 name = "connectors"
-version = "0.4.1"
+version = "0.4.2"
 [dependencies]
 diodes = { git = "JITx-Inc/diodes", version = "0.2.0" }
-
+jsl = { git = "JITx-Inc/jsl", version = "0.10.0" }
```

Comment:
I think `0.10.1` is the absolute latest.

Reply:
updated.